docs/apis-tools/camunda-api-rest/camunda-api-rest-authentication.md

```diff
@@ -4,57 +4,137 @@ title: "Authentication"
+   ```json
+   {
+     "access_token": "<TOKEN>",
+     "expires_in": 300,
+     "refresh_expires_in": 0,
+     "token_type": "Bearer",
+     "not-before-policy": 0
+   }
+   ```
+6. Capture the value of the `access_token` property and store it as your token.
+
+</TabItem>
+
+</Tabs>
+
+## Using a token
+
+Send the captured token as an authorization header in each request: `Authorization: Bearer <TOKEN>`.
+
+For example, to call the Camunda 8 REST API's `/topology` endpoint, use the following command depending on your environment:
+
+<Tabs groupId="using-a-token" defaultValue="saas" queryString values={
+[
+{label: 'SaaS', value: 'saas' },
+{label: 'Self-Managed', value: 'self-managed' },
+]}>
+
+<TabItem value='saas'>
+
+:::tip
+The URL of the Camunda 8 REST API, represented below by the `${ZEEBE_REST_ADDRESS}` variable, can be captured when creating an API client. It can also be constructed as `https://${REGION}.zeebe.camunda.io/${CLUSTER_ID}/`.
+:::
+
+</TabItem>
+
+<TabItem value='self-managed'>
+
+:::tip
+The URL of the Camunda 8 REST API, represented below by the `${ZEEBE_REST_ADDRESS}` variable, is configured in your Self-Managed installation. The default value is `http://localhost:8080/`.
+:::
+
+</TabItem>
+
+</Tabs>
 
 ```shell
-curl -XGET -H'Accept: application/json' -H'Authorization: Bearer <TOKEN>' http://localhost:8080/v2/topology
+curl --header "Authorization: Bearer ${TOKEN}" \
+     ${ZEEBE_REST_ADDRESS}/v2/topology
 ```
 
-### How to obtain the access token
-
-You must obtain a token to use the Camunda 8 REST API. When you create a Zeebe [client](/guides/setup-client-connection-credentials.md), you get all the information needed to connect to Zeebe.
-
-Refer to our guide on [building your own client](../build-your-own-client.md).
-
-The following settings are needed:
-
-| Name                     | Description                                     | Default value      |
-| ------------------------ | ----------------------------------------------- | ------------------ |
-| client id                | Name of your registered client                  | -                  |
-| client secret            | Password for your registered client             | -                  |
-| audience                 | Permission name; if not given use default value | `zeebe.camunda.io` |
-| authorization server url | Token issuer server                             | -                  |
-
-Send a token issue _POST_ request to the authorization server with the following content:
+A successful response would include [information about the cluster](/apis-tools/camunda-api-rest/specifications/get-cluster-topology.api.mdx). For example:
 
 ```json
 {
-  "client_id": "<client-id>",
-  "client_secret": "<client-secret>",
-  "audience": "<audience>",
-  "grant_type": "client_credentials"
+  "brokers": [
+    ...
+  ],
+  "clusterSize": 3,
+  "partitionsCount": 3,
+  "replicationFactor": 3,
+  "gatewayVersion": "8.6.0"
```

Comment:
Idea - Could be fancy here and get a variable for the product version 🤷‍♀️

Reply:
yeah, in general I didn't feel great about these sample API responses. I want to show enough that it's useful, but not so much that people get lost in the details or it's drifting from the real response.